h2o-genmodel/src/main/java/hex/genmodel/easy/EasyPredictModelWrapper.java

```diff
@@ -301,12 +301,29 @@ public EasyPredictModelWrapper(Config config) {
       predictContributions = null;
     }
 
-    CategoricalEncoding categoricalEncoding = config.getUseExternalEncoding() ? 
+    CategoricalEncoding categoricalEncoding = config.getUseExternalEncoding() ?
             CategoricalEncoding.AUTO : m.getCategoricalEncoding();
     Map<String, Integer> columnMapping = categoricalEncoding.createColumnMapping(m);
     Map<Integer, CategoricalEncoder> domainMap = categoricalEncoding.createCategoricalEncoders(m, columnMapping);
 
-    rowDataConverter = RowDataConverterFactory.makeConverter(m, columnMapping, domainMap, errorConsumer, config);
+    final RowtoRawDataConverterProvider rowtoRawDataConverterProvider = getRawDataConverterProviderForGenmodel(m);
```

Comment:
Can we have a `makeConverterFactory` on MojoModel, `makeConvert` method can then be defined on the mojo level with appropriate non-public visibility.

I would go for a simpler solution here.